scripts/percona-release.sh

```diff
@@ -10,7 +10,7 @@ if [[ $(id -u) -gt 0 ]]; then
 fi
 #
 ALIASES="ps56 ps57 ps80 psmdb34 psmdb36 psmdb40 psmdb42 pxb80 pxc56 pxc57 pxc80 ppg11 ppg11.5 ppg11.6 ppg11.7 ppg12 ppg12.2 pdmdb4.2 pdmdb4.2.2 pdmysql8.0.18 pdmysql8.0"
-COMMANDS="enable enable-only setup disable"
+COMMANDS="-y enable enable-only setup disable"
```

Comment:
the noninterective mode is needed for setup only.

Reply:
Fixed